test/rpc-server/Makefile

```diff
@@ -0,0 +1,6 @@
+test:
+	python3 runTests.py
```

Comment:
Unfortunately we need other targets, too: `build-test` and `test-k` are [defined via `include.mk`](https://github.com/runtimeverification/haskell-backend/blob/master/test/include.mk#L157) and used by [scripts/integration-k.sh in Jenkins](https://github.com/runtimeverification/haskell-backend/blob/master/scripts/integration-k.sh#L25)